webcompat/static/js/lib/models/issue.js

```diff
@@ -91,13 +91,11 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var el = body;
-    el = $("strong:contains('Description')", el);
-    if (el[0]) {
-      var description = el[0].nextSibling.textContent;
-      description = description.replace(": ", "");
-      return description != null ? description.slice(0, 74) : null;
-    }
+    var div = document.createElement("div");
+    div.innerHTML = body;
+    var regex = /Description:(.*)+\n/gm;
+    var description = regex.exec(div.textContent);
+    return description != null ? description[1].slice(0, 74) : null;
```

Comment:
Since you're editing this code, can we return a more useful string in the off-chance this doesn't match (rather than null), something like "FIXME - No description found"?

Reply:
@miketaylr Probably not, because this will break this part of the code. 

https://github.com/webcompat/webcompat.com/blob/6ba66d1667646fec75f583e81c570eb5ab201f48/webcompat/static/js/lib/models/issue.js#L110

We handled the `null` with the default title as we were before.

Reply:
Ah, OK. Makes sense.